src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs

```diff
@@ -126,6 +128,43 @@
                                  :else elem))
                              (:data props))}])])})
 
+(react/defc OperationDialog
```

Comment:
Intellij says `react/defc cannot be resolved`. Maybe I also need to import https://github.com/dmohs/react-cljs?